cybersixgill_actionable_alerts/datadog_checks/cybersixgill_actionable_alerts/check.py

```diff
@@ -0,0 +1,143 @@
+import json
+import os
+from datetime import datetime, timedelta
+
+from requests.exceptions import ConnectionError, HTTPError, Timeout
+from sixgill.sixgill_actionable_alert_client import SixgillActionableAlertClient
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+Channel_Id = "d5cd46c205c20c87006b55a18b106428"
```

Comment:
Just curious what is this channel id? Should this be configurable?

Reply:
The channel_id is used by Cybersixgill to monitor the intake of alerts by platform. It was intentionally hardcoded, we don't want user to have access to channel_id